.schema-version

```diff
@@ -1 +1 @@
-v1.2.0
\ No newline at end of file
+v2.1.0
```

Comment:
Is this going to cause a breaking change?

Reply:
Yes, it is. (But it is mostly indirect sign)
The main source of understanding is [changesets](https://changesets-docs.vercel.app/) files.